helptext.txt

```diff
@@ -16,9 +16,9 @@
 **标识**是用于唯一识别存储设备的*签名*。签名
 包括设备型号、序列号、Linux 设备 ID 和设备大小。
 
-**温度**直接从设备读取。您可以在 [显示首选项] (Settings/DisplaySettings)
-页面配置要使用的单位。我们不读取降速硬盘驱动器的温度，
-因为这通常会使它们加速旋转，所以改为显示 `*` 符号。我们还会显示 `*` SSD 和闪存设备的符号，
+**温度**直接从设备读取。您可以在 [设置/显示首选项] (Settings/DisplaySettings)
+页面配置要使用的单位。我们不读取休眠硬盘驱动器的温度，
+因为这通常会使它们重新激活，所以改为显示 `*` 符号。我们还会显示 `*` SSD 和闪存设备的符号，
```

Comment:
19-21: Keep original translation please.

Reply:
The displayed path is different, should I change it back to the original text?
Other will back to original text.